web/olga/analytics/models.py

```diff
@@ -202,14 +202,13 @@ def create_students_per_country(cls, worlds_students_per_country):
         for country, count in worlds_students_per_country.iteritems():
             student_amount_percentage = cls.get_student_amount_percentage(count, all_active_students)
 
-            if country != 'null':
+            try:
                 country_alpha_3 = str(pycountry.countries.get(alpha_2=country).alpha_3)
```

Comment:
L206 and L209 have code duplication could you add in your patch small code refactoring